__tests__/app.test.js

```diff
@@ -58,5 +58,49 @@ describe("api/topics ", () => {
 
 })
 
+describe("api/articles/:article_id ", () => {
+    test("GET: 200 response with the requested article object", () => {
+        return request(app)
+        .get("/api/articles/3")
+        .expect(200)
+        .then(({ body }) => {
+           expect(body.article).toMatchObject({
+            article_id: expect.any(Number),
+            title: expect.any(String),  
+            author: expect.any(String),  
+            body: expect.any(String),     
+            topic: expect.any(String),    
+            created_at: expect.any(String), 
+            votes: expect.any(Number),   
+            article_img_url: expect.any(String) 
```

Comment:
Since we know exactly what article 3 is going to be, we can test for the exact values here! This would be more reliable than checking the data type.